PWGCF/FemtoUnited/TableProducer/femtodreamConverter.cxx

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019-2024 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file femtodreamConverter.cxx
+/// \brief converter task for femtoDream to femtoUnited
```

Comment:
As I will say latter, I would suggest `Femto` as the directory for the `Femto` data model which should be the only one so the `United` suffix should not be needed

In that aim the proper `\brief` should be `converter task for femtoDream datamodel to femto datamodel`
And as I said before, this source file should be under the `FemtoDream/TableProducer` directory